src/TemplateStudio/index.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import createSagaMiddleware from 'redux-saga';
+import { createStore, applyMiddleware } from 'redux';
+import Enzyme, { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import Adapter from 'enzyme-adapter-react-16';
+
+import TemplateStudio from './index';
```

Comment:
We want to import the unconnected component here. So in TemplateStudio/index, add `export` to line 30 and then import `{ TemplateStudio }` in this test file. That way, the snapshot we get should be of our component, not a Redux object.
Then we also will not need the store and saga in this test file. And line 25 would change to just `const component = shallow(<TemplateStudio />);` (although you'll probably also have to mock the props it needs and pass them to it.